cmd/erasure-server-pool-decom.go

```diff
@@ -962,8 +962,11 @@ func (z *erasureServerPools) StartDecommission(ctx context.Context, idx int) (er
 		pathJoin(minioMetaBucket, minioConfigPrefix),
 		pathJoin(minioMetaBucket, bucketMetaPrefix),
 	} {
+		var bucketExists BucketExists
 		if err = z.MakeBucketWithLocation(ctx, metaBucket, BucketOptions{}); err != nil {
-			return err
+			if !errors.As(err, &bucketExists) {
```

Comment:
Since `err`'s underlying type has its bucket name filled in by `toObjectErr`, we need this filled in for errors.Is/errors.As comparisons too. 

```suggestion
			if !errors.Is(err, BucketExists{Bucket: metaBucket}) {
```

Reply:
@krisis I already tested this, I only made it type-specific on purpose.  Not based on the actual `value`

Reply:
oops, incorrect comment. `errors.As` doesn't check error value equality. I got the wrong impression due to an incorrect experiment. The change is good as it were._